apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveResourceModal.tsx

```diff
@@ -157,21 +158,23 @@ const MoveResourceContent = withSuspense(
                   </Text>
                 </Flex>
               )}
-              {children.map((child) => {
-                return (
-                  <MoveItem
-                    {...child}
-                    resourceId={child.id}
-                    key={child.id}
-                    onChangeResourceId={() => {
-                      setResourceStack((prev) => [
-                        ...prev,
-                        { ...child, resourceId: child.id },
-                      ])
-                    }}
-                  />
-                )
-              })}
+              {children
+                .filter((c) => isAllowedToHaveChildren(c.type))
```

Comment:
nit/perf: should we update/create the procedure that already filters this from the sql query itself instead of doing it on the frontend? Maybe I will do that in my pagination PR. Affects pagination query results.

Reply:
actually u r right once again, lemme do this fix

Reply:
nvm u do this fix thx